src/main/java/org/jenkinsci/test/acceptance/plugins/credentials/CredentialsPage.java

```diff
@@ -60,7 +70,7 @@ public WebDriver open() {
         WebDriver wd = super.open();
         // wait for default form fields to be present to avoid possible race
         // condition when changing credentials type too fast (happens rarely)
-        waitFor(by.name("_.scope"));
+        waitFor(by.name("_.id"));
```

Comment:
This changes the field to wait for (introduced in [PR #243](https://github.com/jenkinsci/acceptance-test-harness/pull/243) as personal user credentials does not have scope.